e2e/portalicious/pages/RegistrationActivityLogPage.ts

```diff
@@ -27,6 +27,15 @@ class RegistrationActivityLogPage extends BasePage {
     expect(sentMessageText).toBe(message);
     await dropdownButton.click();
   }
+
+  async clickActionDropdown() {
```

Comment:
I've made quite some changes in #6522 that add a "RegistrationBasePage" because these actions will be relevant for all registration pages. Maybe worth seeing how bring these new functions into there.

Reply:
Feels like we need to merge that one first than I can rebase and see :)